dotCMS/src/main/java/com/dotcms/analytics/bayesian/model/AbstractBayesianPriors.java

```diff
@@ -0,0 +1,30 @@
+package com.dotcms.analytics.bayesian.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.immutables.value.Value;
+
+import javax.annotation.Nullable;
+
+
+/**
+ * Single Bayesian Prior data class
+ *
+ * @author vico
+ */
+@Value.Style(typeImmutable="*", typeAbstract="Abstract*")
+@Value.Immutable
+@JsonDeserialize(as = BayesianPriors.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public interface AbstractBayesianPriors {
```

Comment:
Would be nice to have a better documentation about this class and represented values